finetune/target_models/document_labeling.py

```diff
@@ -0,0 +1,140 @@
+from finetune.target_models.sequence_labeling import SequenceLabeler
+
+def get_context(document, dpi_norm):
+    context = []
+    for page in document:
+        for token in page["tokens"]:
+            pos = token["position"]
+            offset = token["doc_offset"]
+            if dpi_norm:
+                dpi = page["pages"][0]["dpi"]
+                x_norm = 300 / dpi["dpix"]
```

Comment:
can we add a quick comment mentioning that the base model was trained on positions from pages with 300 dpi?